### PR TITLE
Modified a file and added new

### PR DESCRIPTION
**Pull request by ssan9**
Modified README.md by added Hello World in it and added foo.html with Hi in it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Trial
 GitHub practice
+Hello World
```
Comment (nktand1485) [inline README.md:3]: It's good enough

File: foo.html [hunk 2]
```diff
@@ -0,0 +1 @@
+Hi
```
Comment (nktand1485) [inline foo.html]: It would be better if we change it to hello world

Comment (ssan9) [reply foo.html]: Ok. I'll change it

Comment (ssan9) [reply foo.html]: Done

**Review by nktand1485 — APPROVED** [review]

**Pull request merged**